src/controllers/bookings.js

```diff
@@ -0,0 +1,122 @@
+const Booking = require("../models/booking");
+const { sequalize } = require("../configs/mysqldb");
+const { setCache, getCache } = require("../configs/cacheConfig");
+
+const AvailableSeats = {
```

Comment:
Could have initialised variables with the help of loop. Manual code is not required.